db/inst/import-try/01.global.subset.R

```diff
@@ -0,0 +1,36 @@
+# 1. Global subset
+library(data.table)
+library(bit64)
+message("Loading TRY data...")
```

Comment:
@dlebauer Yes, but I still don't fully know what they do -- `message` is base R and can be controlled neatly from `knitr` and is handled by base R functionality. But once this gets more directly implemented in PEcAn, there's no reason these can't be changed to `logger` calls.